src/main/java/org/ibp/api/rest/labelprinting/StudyEntriesLabelPrinting.java

```diff
@@ -87,7 +85,7 @@ LabelsNeededSummaryResponse transformLabelsNeededSummary(final LabelsNeededSumma
 	@Override
 	OriginResourceMetadata getOriginResourceMetadata(final LabelsInfoInput labelsInfoInput, final String programUUID) {
 		final StudyDetails study = this.studyDataManager.getStudyDetails(labelsInfoInput.getStudyId());
-		final String tempFileName = study.getStudyName().concat("_").concat(StudyEntriesLabelPrinting.ORIG_FINAL_NAME);
+		final String tempFileName = study.getStudyName().concat(LabelPrintingFieldUtils.UNDERSCORE).concat(StudyEntriesLabelPrinting.ORIG_FINAL_NAME);
```

Comment:
@cuenyad I think this underscore is not directly related with the one in Utils, say that you want to change the combined key, then the file format will also be changed. Please revert this change

Reply:
done!